### PR TITLE
add info about user session timeout

### PR DESCRIPTION
**Pull request by lemingos**
Added info

### CODE REVIEW

File: marketplace_builder/views/pages/tutorials/partner-portal/updating-instance-configuration.liquid [hunk 1]
```diff
@@ -92,12 +92,24 @@ You can enter multiple emails, comma-separated.
 "test_email": "sales@platform-os.com,developer@platform-os.com,support@platform-os.com"
 ```
 
+##### timeout_in_minutes
+
+This settings allows for changing user session duration. If not set, default timeout is 1 year. If you want log out users after 15 minutes of inactivity set it as in the example below:
+
+**Default value:** `1 year`
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/partner-portal/updating-instance-configuration.liquid]: image

**Review by pavelloz — CHANGES_REQUESTED** [review]
if units is minutes, default value is not in years

File: marketplace_builder/views/pages/tutorials/partner-portal/updating-instance-configuration.liquid (hunk 1 above)
Comment (diana-lakatos) [reply marketplace_builder/views/pages/tutorials/partner-portal/updating-instance-configuration.liquid]: I updated the default value. Different converters gave me a different result for 1 year in minutes, so let's double-check if this is the actual value we use. Thank you!

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited, ready to be merged after double-checking default value in minutes. Thank you!

**Pull request merged**